docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -135,3 +135,12 @@ in Spark WebUI below.
 
 ![locality]({{site.data.img.screenshot_datalocality_tasklocality}})
 
+### Running Spark on YARN
+
+When a Spark job is run on YARN, Spark launches its executors without taking data locality into account.
+Spark will then correctly take data locality into account when deciding how to distribute tasks to its
+executors. For example, if `host1` contains `blockA` and a job using `blockA` is launched on the yarn
+cluster with `--num-executors=1`, Spark might place the only executor on `host2` and have poor locality.
+However, if `--num-executors=2` and executors are started on `host1` and `host2`, Spark will be smart
+enough to prioritize placing the job on host1. To maximize the amount of locality your Spark jobs attain,
+you should use as many executors as possible, ideally 1-2 executors per node in the YARN cluster.
```

Comment:
`1-2` isn't special, we just want to cover as many nodes as possible. I've changed it to say at least one executor per node

Alluxio should be deployed on all computation nodes, I've added this